editor/components/writing-flow/index.js

```diff
@@ -201,6 +202,25 @@ class WritingFlow extends Component {
 		}
 
 		if ( ! isNav ) {
+			const activeElement = document.activeElement;
+
+			// Set right before the meta+a combination can be pressed.
```

Comment:
Instead of "right before", I would write "immediately before". It's clearer and a bit easier to parse (I had to re-read this sentence to grok it.)